uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -64,6 +64,7 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
   }
 
   @Test def skipNonExistentBranches_toggleListingCommits_slideOutRoot(): Unit = {
+    intelliJ.machete.runJs(s"project.openGitMacheteTab()")
```

Comment:
Maybe drop a comment on why this was needed... if there's a clear reason, at least

Reply:
Also, nit: `scala.StringContext#s` not needed since no interpolation happens inside

Reply:
> Maybe drop a comment on why this was needed... if there's a clear reason, at least

I do not think that any comment in code is necessary.
Furthermore, I see no right place for such a comment. 
However, I will give an explanation here.

We are talking about this fragment of code:
```
@Test def discoverBranchLayout(): Unit = {
    // When model is refreshed and machete file is has not been modified for a long time, then discover suggestion should occur
    setLastModifiedDateOfMacheteFileToEpochStart()
    intelliJ.machete.runJs("project.openGitMacheteTab()")
    //...
  }
```

The point is that to test the discover suggestion functionality we must prepare the test (set the modified date of the file) before the check (of the time last) occurs, obviously.

The "problem" that appeared is that the check happens on the component creation (at the moment when intellij content manager "adds" (displays) `GitMachetePanel` actually). Hence we must prepare for the test before the first tab opening. To do this I had to extract the tab opening from before each method.